demisto_sdk/commands/validate/tests/IM_validators_test.py

```diff
@@ -41,6 +44,34 @@ def test_ImageExistsValidator_is_valid_image_path():
     )
 
 
+def test_ImageTooLargeValidator_is_valid():
+    """
+    Given:
+    content_items with 2 integrations:
+        - One integration with an image that is not in a valid size.
+        - One integration with an image that is in a valid size.
+
+    When:
+        - Calling the ImageTooLargeValidator is_valid function.
+
+    Then:
+        - Make sure the right amount of integration image path failed, and that the right error message is returned.
+        - Case 1: Should fail.
+        - Case 2: Shouldn't fail.
+    """
+    content_items = [
```

Comment:
Please change the unit tests to use pytest's parametrize feature for clarification which
test should pass/fail